### ISSUE TITLE
Submit to F-droid

### ISSUE THREAD

**Issue by Rustmilian** [liftoff-app/liftoff#47]
[Submitting to F-droid](https://f-droid.org/docs/Submitting_to_F-Droid_Quick_Start_Guide/) would increase discoverability of your app.

**Comment by zachatrocity** [comment]
This is my next focus. Getting it on play store was first. While that's in Google review I'll get F-Droid working

**Comment by weiss-johannes** [comment]
Submit to the  [izzysoft](https://apt.izzysoft.de/fdroid/index/info) repo would be a nice addition because Lemmy and this App make fast progress and the official f-droid is very slow in updating the app.

**Comment by zachatrocity** [comment]
Yeah honestly F-Droid kinda sucks I'll say it. 

Obtainium is a much better solution.

The app will be on Google play early access soon anyways. It's in review.

I'll get to this eventually but it's not my top priority right now

**Comment by zell-mbc** [comment]
From my experience with my own little app...once you got things set up for F-Droid and Izzy it's fire and forget.
And the delays aren't as bad s they used to be…
I'll be happily waiting for LiftOff to show up in F-Droid

**Comment by zachatrocity** [comment]
There's quite a few people that feel the same way as you and I know I'm in the minority.

I'll get it up there soon, just gotta read through their docs

**Comment by Generator** [comment]
Made a request on IzzyOnDroid.  
Can follow the issue here https://gitlab.com/IzzyOnDroid/repo/-/issues/407

**Comment by tcely** [comment]
I found the MR for this at: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/13291

**Comment by Darin755** [comment]
Any update to this? I still don't see the app in the main F-droid repo

**Comment by mpeter50** [comment]
It seems the app was removed from Izzy's repo

**Comment by Darin755** [comment]
> It seems the app was removed from Izzy's repo

You might want to open a new issue

**Comment by mpeter50** [comment]
I dont think it would make a difference. Izzy's repo is like a staging area to F-droid for new apps, with less strict requirements, until they can complete publishing the app normally to F-droid main. He offers that service in good faith, assuming that its use is only temporary. He removes apps from his own repo if the app did not make progress on a normal F-droid release for quite some time.

The main thing to solve here is the normal F-droid relase.
I have noted it here because I have just noticed that its missing, so others can know to not expect to find it there anymore.

**Comment by Darin755** [comment]
That wasn't my point. You are talking about a new issue entirely. It is really up to the owner of this repo but for me personally I don't like issues that are talking about multiple problems.
